src/service-api.ts

```diff
@@ -0,0 +1,29 @@
+import { FastifyPluginAsync } from 'fastify';
+
+import { AdminService } from './db-service';
+import { isAdmin } from './schemas';
+import { TaskManager } from './task-manager';
+
+// Placeholder for options
+// export interface GraaspAdminPluginOptions {
+// }
+
+const plugin: FastifyPluginAsync = async (fastify) => {
+  const {
+    taskRunner: runner,
+  } = fastify;
+  const adminService = new AdminService();
+  const taskManager = new TaskManager(adminService);
+
+  // Check if current user is admin ( OR given user ? )
+  fastify.get(
+    '/admin/current',
+    { schema: isAdmin },
+    async ({ member, log }) => {
+      const task = taskManager.createIsAdminTask(member);
+      return runner.runSingle(task, log);
+    },
+  );
```

Comment:
Probably it's more interesting to have an endpoint for getting the user roles. I don't think we will use an endpoint to know if the user is an admin. 
The way we will use `isAdmin` is from fastify prehook 
https://github.com/graasp/graasp/blob/19ab182e1ad208515e1747acd9c049fbb8d19b92/src/services/items/service-api.ts#L135
So would it be possible to have an importable function `isAdmin()` using the dbService to get the data from the db?